src/py21cmfast/src/HaloBox.c

```diff
@@ -681,7 +681,7 @@ int ComputeHaloBox(double redshift, UserParams *user_params, CosmoParams *cosmo_
         LOG_DEBUG("Gridding %llu halos...",halos->n_halos);
 
         double M_min = minimum_source_mass(redshift,false,astro_params,flag_options);
-        double M_max = M_MAX_INTEGRAL;
+        double M_max;
```

Comment:
This should probably be named to something that makes it clear it's the maximum mass of the *integrated* component of the halo boxes, so that nobody can confuse it with the maximum halo mass overall